blacksauce/src/styles/_base.scss

```diff
@@ -0,0 +1,90 @@
+label {
```

Comment:
The organization of this is a little unstructured. I recommend starting at the top with your `html`, `body`, etc., then dealing with block-level elements, then inline, so roughly like so:
```
html {}

*,
*::before,
*::after {
}

body {}

h1,
h2,
p {}

h1 {}

h2 {}

p {}

q {}

ul {}

label {}

input[type="button"],
input[type="submit"],
input[type="reset"] {}

textarea {}

::placeholder {}

```
I removed the CSS values and properties, just to order the selectors.